### PR TITLE
Update free-programming-books-tr.md

### PR DESCRIPTION
**Pull request by emraher**
r4ds cevirisi eklendi

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
good addition, but be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.

File: books/free-programming-books-tr.md [hunk 1]
```diff
@@ -142,6 +142,7 @@
 
 * [Ekonometriye Yeni Başlayanlar için Kısa bir R Kılavuzu](https://www.github.com/emraher/eybkbrk) - Emrah Er
 * [R ile Programlamaya Giriş ve Uygulamalar (2014)](http://inet-tr.org.tr/inetconf19/sunum/16.pdf) - Mustafa Gökçe Baydoğan, Berk Orbay, Uzay Çetin (PDF)
+* [R for Data Science](http://tr.r4ds.hadley.nz/) - Garrett Grolemund, Hadley Wickham
```
Comment (davorpa) [inline books/free-programming-books-tr.md:145]: - Alphabetize
- We love give also credits to traducers, so... add them to creators list using role anotations :wink:

```suggestion
* [R for Data Science](http://tr.r4ds.hadley.nz/) - Garrett Grolemund, Hadley Wickham, `trad.:` İsmail Bekar, `trad.:` Nurbahar Usta, `trad.:` Bilgecan Şen
* [R ile Programlamaya Giriş ve Uygulamalar (2014)](http://inet-tr.org.tr/inetconf19/sunum/16.pdf) - Mustafa Gökçe Baydoğan, Berk Orbay, Uzay Çetin (PDF)
```

**Review by davorpa — CHANGES_REQUESTED** [review]
As Eric requests...

**Review by davorpa — APPROVED** [review]
LGTM.

Well seen this ending slash :rocket: 4874524757226ed6fcd627cf43f5a470da5eb847

**Pull request merged**